README.md

```diff
@@ -2,42 +2,12 @@
 
```

Comment:
There are some changes in master to the README that fix up some deployment-manager -> helm changes (notables: badges need to be changed). I think if you rebase you'll pick up most of those.